frame/system/src/lib.rs

```diff
@@ -860,12 +860,19 @@ impl<T: Trait> Module<T> {
 		<ExtrinsicsRoot<T>>::put(txs_root);
 
 		if let InitKind::Full = kind {
+			sp_io::storage::accumulator_commit(&Events::<T>::storage_value_final_key()[..]);
```

Comment:
could clean these imports prior to merge, but not a big deal.

Reply:
What imports?